environs/testing/tools.go

```diff
@@ -291,207 +291,24 @@ var (
 	V100u32 = version.MustParseBinary("1.0.0-ubuntu-arm64")
 	V100p   = []version.Binary{V100u64, V100u32}
 
-	V100c64 = version.MustParseBinary("1.0.0-centos-amd64")
-	V100c32 = version.MustParseBinary("1.0.0-centos-arm64")
-	V100q   = []version.Binary{V100c64, V100c32}
-	V100all = append(V100p, V100q...)
-
 	V1001    = version.MustParse("1.0.0.1")
 	V1001u64 = version.MustParseBinary("1.0.0.1-ubuntu-amd64")
-	V100Xall = append(V100all, V1001u64)
 
 	V110    = version.MustParse("1.1.0")
 	V110u64 = version.MustParseBinary("1.1.0-ubuntu-amd64")
 	V110u32 = version.MustParseBinary("1.1.0-ubuntu-arm64")
 	V110p   = []version.Binary{V110u64, V110u32}
 
-	V110c64 = version.MustParseBinary("1.1.0-centos-amd64")
-	V110c32 = version.MustParseBinary("1.1.0-centos-arm64")
-	V110c   = []version.Binary{V110c64, V110c32}
-	V110all = append(V110p, V110c...)
-
 	V120    = version.MustParse("1.2.0")
 	V120u64 = version.MustParseBinary("1.2.0-ubuntu-amd64")
 	V120u32 = version.MustParseBinary("1.2.0-ubuntu-arm64")
 	V120all = []version.Binary{V120u64, V120u32}
 
-	V1all = append(V100Xall, append(V110all, V120all...)...)
+	V1all = append(V100p, append(V110p, V120all...)...)
 
 	V220    = version.MustParse("2.2.0")
 	V220u32 = version.MustParseBinary("2.2.0-ubuntu-arm64")
 	V220u64 = version.MustParseBinary("2.2.0-ubuntu-amd64")
 	V220all = []version.Binary{V220u64, V220u32}
 	VAll    = append(V1all, V220all...)
 )
-
-type BootstrapToolsTest struct {
```

Comment:
What's the reason for removing these?
If anything, shouldn't we be up[dating them to use DefaultBase not DefaultSeries?

Reply:
This was a flyby, these were unused

Reply:
They have been unused since at least 2.9! So I reckon at this point removing them is the correct solution

This const was added 11 years ago, transferred over from another place and renamed. They were used by `TestFindBootstrapTools`, a test which no longer exists
https://github.com/juju/juju/commit/5050b851e63955dd8f4a43444089db7ffb7cf07d